### DISCUSSION TITLE
Unable to create collision module

### DISCUSSION THREAD

**Discussion by ZhihongLiao** [InfinyTech3D/SofaUnity#121] [Help - Using SofaUnity]
When I use the sofaunity in Unity, I wanna create a sofasphere but it shows:
![屏幕截图 2024-09-07 002028](https://github.com/user-attachments/assets/1bd7a8d2-8410-40f0-ac9b-e2134ba04709)
![屏幕截图 2024-09-07 002004](https://github.com/user-attachments/assets/a49aa1a5-c888-4d32-9c4a-964801a0088b)
And when I use the SofaSphereCollisionModel, it shows:
![屏幕截图 2024-09-07 002332](https://github.com/user-attachments/assets/8496acff-dc2b-4e13-a9e9-99930d6717fd)
Please help me, thank you. I use the sofaunity(v23.12).

**Comment by epernod** [comment]
Hi,

could you copy past your .scn file here? 
Thanks

**Reply by ZhihongLiao** [reply]
I don't have .scn file in my scene. So, must I import a .scn file after I create a sofaconext?
These are my  all .scn files.
![屏幕截图 2024-09-09 093619](https://github.com/user-attachments/assets/2b75f77b-cb20-4905-8c95-855e245c3e6b)
But these seem to be brought in the sofaunity package.

**Reply by epernod** [reply]
Hi,

yes your simulation inside Unity need to be based on a SOFA .scn file. See this page: https://infinytech3d.com/feature_scene-parsing/

Best regards,

**Reply by ZhihongLiao** [reply]
How do I creat a scn file？

**Reply by ZhihongLiao** [reply]
I don't know where I can get the scn file or creat a scn file.

**Reply by MingSongJiang** [reply]
直接复制自带的.scn文件

**Reply by ZhihongLiao** [reply]
啊？我在上面发了一些包里有的scn文件，我随便复制一个？能行吗？

**Reply by ZhihongLiao** [reply]
> 直接复制自带的.scn文件
啊？我在上面发了一些包里有的scn文件，我随便复制一个？能行吗？

**Reply by ZhihongLiao** [reply]
> 直接复制自带的.scn文件
不知道该复制哪些

**Reply by epernod** [reply]
Thanks for answering. Please keep English for all users.

**Reply by ZhihongLiao** [reply]
I do not solve this problem. Sorry.

**Reply by ZhihongLiao** [reply]
> Thanks for answering. Please keep English for all users.

I do not solve this problem. Sorry.
